cmd/bootstrapper/uninstall_kubelet.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"flag"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/windows-machine-config-bootstrapper/pkg/bootstrapper"
+)
+
+var (
+	// uninstallKubeletCmd describes the uninstall-kubelet command
+	uninstallKubeletCmd = &cobra.Command{
+		Use:   "uninstall-kubelet",
+		Short: "Uninstalls the kubelet service from the Windows node",
+		Long:  "Uninstalls the kubelet service from the Windows node",
```

Comment:
`Stops and removes the kubelet service` maybe?

Windows documentation seems to use the term `remove` when it comes to services and not `uninstall`.
Dont think mentioning the Windows node is necessary as this is run directly on the Windows VM.